src/web/data/userMapData.json

```diff
@@ -1,2538 +1,2630 @@
-      "Medical College of Wisconsin",
-      "Medical University of the Americas",
-      "Mercer Middle School",
-      "Mercer University",
-      "Merrimack College",
-      "Methuen High School",
-      "Methodist University",
-      "Metropolitan Community College",
-      "Metropolitan State University of Denver",
-      "MGH Institute of Health Professions",
-      "Miami Dade College",
-      "Miami International University of Art and Design",
-      "Miami University",
-      "Miami University Regionals",
-      "Michigan State University",
-      "Michigan Technological University",
-      "Mid Pacific Institute",
-      "Mid-Del Technology Center",
-      "Middle Creek High School",
-      "Middlesex Community College",
-      "Midland University",
-      "Midway High School",
-      "Midwives College of Utah",
-      "Miles College",
-      "Miller Career & Technical Center",
-      "Mills College",
-      "Millsaps College",
-      "Milwaukee Area Technical College",
-      "Milwaukee School of Engineering",
-      "Mississippi State University",
-      "Missouri Southern State University",
-      "Missouri State University",
-      "Missouri Valley College",
-      "MIT",
-      "Mountain View High School",
-      "Montana State University",
-      "Montclair High School",
-      "Montclair State University",
-      "Monte Vista Christian High School",
-      "Monterey Peninsula College",
-      "Montgomery College",
-      "Moorpark College",
-      "Moreno Valley High School",
-      "Morgan County Charter School System",
-      "Morgan County Middle School",
-      "Morgan State University",
-      "Morningside College",
-      "Morton Ranch High School",
-      "Mounds View Public Schools",
-      "Mount Mercy University",
-      "Mount Saint Mary's University",
-      "Mount St. Mary's College",
-      "Mount Vernon Nazarene University",
-      "Mount Zion High School",
-      "Mountain Home Public School",
-      "Mountain View College",
-      "Muhlenberg College",
-      "Myrtle Grove Middle School",
-      "N2N Services",
-      "Naaman Forest High School",
-      "Nash Rocky Mount Public Schools",
-      "Navarro College",
-      "Navasota High School",
-      "Nazareth College",
-      "NC A&T State University",
-      "NC State University",
-      "Nevada State College",
-      "New Bern High School",
-      "New Brunswick Health Sciences Technology High School",
-      "New England Insitute of Technology",
-      "New Fairfield High School",
-      "New Jersey City University",
-      "New Jersey Institute of Technology",
-      "New Mexico State University",
-      "New Prairie United School Corporation",
-      "New Tech Odessa High School",
-      "New York Institute of Technology",
-      "New York University",
-      "Newbury Park High School",
-      "Newport News Schools",
-      "Newton Conover City Schools",
-      "Nichols College",
-      "Noncommsioned Officer
```

Comment:
The indentation for JSON file is wrong? Should be 2 spaces?

Reply:
Correct. beautified to indentation of 1 space as it's in previous commits of this file.